test/inspector/test-async-hook-setup-at-inspect.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+const common = require('../common');
+common.skipIfInspectorDisabled();
+common.skipIf32Bits();
+common.crashOnUnhandledRejection();
+const { NodeInstance } = require('../inspector/inspector-helper.js');
+const assert = require('assert');
+
+// Even with --inspect, the default async call stack depth is 0. We need a
+// chance to call Debugger.setAsyncCallStackDepth *before* activating the timer
+// for async stack traces to work.
```

Comment:
Wow, good catch, it didn't occur to me that async call stacks depths may be zero and therefore there will be no async stack recorded.